### PR TITLE
Add python3-tikzplotlib-pip

### PR DESCRIPTION
**Pull request by Wojcik98**
Please add the following dependency to the rosdep database.

## Package name:

tikzplotlib

## Package Upstream Source:

https://pypi.org/project/tikzplotlib/

## Purpose of using this:

This package is used to convert matplotlib figures to PGFPlots format for native inclusion into LaTeX or ConTeXt documents.

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://pypi.org/project/tikzplotlib/
- Ubuntu: https://packages.ubuntu.com/
  - https://pypi.org/project/tikzplotlib/
- macOS: https://formulae.brew.sh/
  - https://pypi.org/project/tikzplotlib/

### CODE REVIEW

**Review by nuclearsandwich — APPROVED** [review]

**Comment by Wojcik98** [comment]
Hi @nuclearsandwich, do you know why the Github checks are still not finished? It's been a while.

**Comment by nuclearsandwich** [comment]
> Hi @nuclearsandwich, do you know why the Github checks are still not finished? It's been a while.

PRs from first time contributors require approval before they are run. I approved the first round but I had not revisited after your merge to resynchronize the PR with master. I've approved the run now. Once a PR of yours has landed, you will no longer be considered a first time contributor and checks on your PRs will run directly.

**Review by mjeronimo — APPROVED** [review]

**Pull request merged**
